src/js/builtins/ConsoleObject.ts

```diff
@@ -289,28 +292,35 @@ export function createConsoleConstructor(console: typeof globalThis.console) {
     } = options;
 
     if (!stdout || typeof stdout.write !== "function") {
-      // throw new ERR_CONSOLE_WRITABLE_STREAM("stdout");
-      throw new TypeError("stdout is not a writable stream");
+      const err = new TypeError("stdout is not a writable stream");
+      err.code = "ERR_CONSOLE_WRITABLE_STREAM";
```

Comment:
Please add these to ErrorCode.ts

```suggestion
      throw $ERR_CONSOLE_WRITABLE_STREAM("stdout is not a writable stream");
```